src/readabilityScore/Index.java

```diff
@@ -0,0 +1,34 @@
+package readabilityScore;
+
+public enum Index {
+    // Corresponds to Automated readability index
+    ARI {
+        @Override
+        IndexCalculator getCalculator() {
+            return new AutomatedReadabilityIndexCalculator();
+        }
+    },
```

Comment:
It's okay to override the methods right inside of the enum instances. But unfortunately, it takes a lot of space to override them...

To simplify that we could have a single method instead:
```
    public IndexCalculator getCalculator() {
        switch (this) {
            case ARI: return new AutomatedReadabilityIndexCalculator();
            case FK: return new FleschKincaidIndexCalculator();
            case SMOG: return new SMOGIndexCalculator();
            case CL: return new ColemanLiauIndexCalculator();
        }
        return null;
    }
```

It looks a bit shorter and makes it easier to read the enum instances, wdyt?